specification/dartLangSpec.tex

```diff
@@ -8862,16 +8888,16 @@ \subsubsection{Function Expression Invocation}
 \begin{itemize}
 \item \code{$im$.isMethod} evaluates to \code{\TRUE{}}.
 \item \code{$im$.memberName} evaluates to the symbol \code{\#call}.
-\item \code{$im$.positionalArguments} evaluates to an unmodifiable list with the values
-resulting from evaluation of
-\code{<Object>[$a_1, \ldots,\ a_n$]}.
+\item \code{$im$.positionalArguments} evaluates to an unmodifiable list with the objects
```

Comment:
-> containing the objects

(like below).

Reply:
Done.